app/src/main/java/org/dhis2/usescases/programEventDetail/ProgramEventDetailViewModel.kt

```diff
@@ -43,6 +60,63 @@ class ProgramEventDetailViewModel(
     val shouldNavigateToEventDetails: SharedFlow<String>
         get() = _shouldNavigateToEventDetails
 
+    private val _navigationBarUIState = mutableStateOf(NavigationBarUIState<NavigationPage>())
+    val navigationBarUIState: State<NavigationBarUIState<NavigationPage>> = _navigationBarUIState
+
+    init {
+        viewModelScope.launch { loadBottomBarItems() }
+    }
```

Comment:
I would recommend not to do any initialization in a viewModel's `init{}` because it can cause weird behavior on config changes and also, it will be harder to test.

Maybe you could use flows and emit the changes of the bar's items